webapp/lib/controller.dart

```diff
@@ -90,6 +99,23 @@ void initUI() {
   );
 }
 
+void checkNeedsReplyMetricsPipelineDataFlow() {
+  var sortedNeedsReplyDataList = List.from(needsReplyDataList);
+  sortedNeedsReplyDataList.sort((d1, d2) => d1.datetime.compareTo(d2.datetime));
+  var lastNeedsReplyEntry = sortedNeedsReplyDataList.where((entry)=>
+      entry.project == view.contentView.projectSelectorView.selectedProject).last;
+
+  var now = new DateTime.now();
+  var lastUpdateTimeDiff =  now.difference(lastNeedsReplyEntry.datetime).inHours;
```

Comment:
nit: rename to `lastUpdateTimeDiffInHours`

Reply:
Also, does `.inHours` return an `int` or `double`? it would really help here if the type of this variable was listed explicitly

Reply:
Returns an int. I'll explicitly state the type